modules/ROOT/partials/configuration/forced_root_block.adoc

```diff
@@ -18,6 +18,6 @@ Type : `+String+`
 ----
 tinymce.init({
   selector: 'textarea',  // change this value according to your HTML
-  forced_root_block : 'p'
+  forced_root_block: 'p'
```

Comment:
`p` is the default here so we should use something else. We also should add this between the type and example:
```
Default Value: `+'p'+`
```